PC/launcher.c

```diff
@@ -1412,22 +1412,29 @@ show_help_text(wchar_t ** argv)
 Python Launcher for Windows Version %ls\n\n", version_text);
     fwprintf(stdout, L"\
 usage:\n\
-%ls [launcher-args] [python-args] script [script-args]\n\n", argv[0]);
+%ls [launcher-args] [python-args] [script [script-args]]\n\n", argv[0]);
     fputws(L"\
 Launcher arguments:\n\n\
--2     : Launch the latest Python 2.x version\n\
--3     : Launch the latest Python 3.x version\n\
+-2     : Launch the latest* Python 2.x version\n\
+-3     : Launch the latest* Python 3.x version\n\
```

Comment:
I think we can do without the footnote marker, as the comment below applies to practically all possibilities (and the text itself explains which cases are not covered).

Reply:
Addressed!